sources/vega/commands/v1/validator_commands.proto

```diff
@@ -82,8 +82,6 @@ message KeyRotateSubmission {
   uint32 key_number = 1;
```

Comment:
Yeah, you are right.. I though about it yesterday and then I went through the code and convinced myself that hash would work (don't ask me how I concluded it - it was late I guess..), but looking at it again it will need to be public key.. Encoded to hex but not hashed.